deploy/utils.sh

```diff
@@ -13,22 +16,25 @@ wait_for_it() {
   local timeout=$1
   local spacer=5
   shift
-  if ! [ $timeout = '-1' ]; then
+  if ! [ "${timeout}" = '-1' ]; then
     local times_to_run=$((timeout / spacer))
 
-    #echo "Waiting for '$@' up to $timeout s"
     for i in $(seq $times_to_run); do
-      eval $*  && return 0
+      if cmd_output=$(eval "$@") ;
```

Comment:
Avoid `eval`.  Instead, you can de-structure this function's arguments into `cmd` and `args`, and then do:
```sh
      if cmd_output=$("$cmd" "${args[@]}") ; then
```

Reply:
it's not that simple. please look for usages of `wait_for_it`.

Reply:
Fair enough.  Should probably be changed everywhere forcing code to be wrapped in functions, but I won't ask you to do that :)